pkg/features/controller.go

```diff
@@ -120,3 +121,7 @@ func (f *FeatureController) Start(ctx context.Context, feature *v1.Feature) erro
 
 	return nil
 }
+
+func (f *FeatureController) Priority() int {
+	return f.PriorityWeight
```

Comment:
Why do we need a priority?

Reply:
want to have better control over the order of features